docs/troubleshooting.md

```diff
@@ -0,0 +1,217 @@
+
+# Troubleshooting:
+
+_For commonly asked questions and their answers, you can refer to the [FAQ](./faq.md)_
+
+**source{d} CE** is still in beta stage, so some features are still missing or
```

Comment:
Let's remove the mention to beta here, it will be easy to forget about it when we release the final one. And this already is visible when you download the version.